src/Analyzers/CSharp/Tests/ConvertNamespace/ConvertToFileScopedNamespaceAnalyzerTests.cs

```diff
@@ -787,8 +784,8 @@ public async Task TestConvertToFileScopedSingleLineNamespace1()
 [|namespace N|] { class C { } }
 ",
                 FixedCode = @"
-namespace $$N; 
-class C { } ",
+namespace $$N; class C { } 
```

Comment:
another improvement. the original code was on the same line as the namespace.  we preserve that.